git-hub

```diff
@@ -81,21 +137,325 @@ def push():
+        path_prs = pjoin(path_github, 'pull-requests.toml')
+        f = open(path_prs, "r")
+        pr_dict = toml.load(f)      # fetches toml file and creates a dictionary
+        open_dict = pr_dict['open pull requests']
+        closed_dict = pr_dict['closed pull requests']
+    except (OSError, IOError) as e:
+        # if pull-requests.toml hasnt been created yet calls sync and then reties to fetch
+        sync()
+        get_info(pr_num)
+
+    open_PR = "O"
+    key, pr_dictionary = find_in_dictionary(pr_num, open_dict)
+    if(pr_dictionary is None):
+        open_PR = "C"
+        key, pr_dictionary = find_in_dictionary(pr_num, closed_dict)
+    if(pr_dictionary is None):
+        click.echo("Could not find PR #{}. Run 'git hub sync' and try again.".format(pr_num))
+    else:
+        reviewers = "None"
+        if(pr_dictionary['reviewers'] is not ""):
+            reviewers = pr_dictionary['reviewers']
+        assignee = "None"
+        if(pr_dictionary['assignee'] is not ""):
+            assignee = pr_dictionary['assignee']
+        milestone = "None"
+        if(pr_dictionary['milestone'] is not ""):
+            milestone = pr_dictionary['milestone']
+        labels = "None"
+        if(pr_dictionary['labels'] is not ""):
+            labels = pr_dictionary['labels']
+        click.echo('{0} \x1b[0;32;40m {1} \x1b[0m {2}/{3} {4}'.format(key, open_PR, pr_dictionary['user'], pr_dictionary['branch'], pr_dictionary['comment']))
+        click.echo('-Labels: {0}  -Reviewers: {1}  -Assignees: {2}  -Milestones: {3}'.format(labels, reviewers, assignee, milestone))
+
+
+def print_in_order(sort, dict):
+    """
+    Prints items in q sorted by order given by sort
+
+    Parameters
+    ----------
+    sort : string
+        sort in increasing or decreasing order
+    q : PriorityQueue
+        items to be sorted, currently in increasing order
+    """
+    if(sort.startswith('i')):
+        sorted_list = sorted(dict)
+        for x in sorted_list:
+            click.echo("{0} : {1}".format(dict[x], x[:10]))
+    else:
+        sorted_list = sorted(dict, reverse=True)
+        for x in sorted_list:
+            click.echo("{0} : {1}".format(dict[x], x[:10]))
+
+
+def find_match(open_or_closed, list_of_dictionaries, keyword, user, comment, number, branch, label):
+    """Helper function that finds all matches in the given dictionary that fits the specified parameters
+
+    Parameters
+    ----------
+    open_or_closed : boolean
+        Whether looking in closed PR or open PR
+    list_of_dictionaries : dictionary
+        pull-requests.toml information mapping PR number to info
+    keyword : string
+        Searches if any part of user, branch, comment, or number match
+    user : string
+        Search by PR username
+    comment : string
+        Search by PR comment
+    number : string
+        Search by PR number
+    branch : string
+        Search by PR branch
+    appeared_before : boolean
```

Comment:
This seems to be a return that is documented under Parameters.  Also second return is undocumented.